tests/export/html/test_markup_compatibility.py

```diff
@@ -91,3 +91,71 @@ def test_textbox_with_content_outside_of_textbox(self):
             </p>
         '''
         self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_only_text(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <r>
+                                <t>CCC</t>
+                            </r>
+                        </Fallback>
+                    </AlternateContent>
+                    <t>DDD</t>
+                </r>
+                <r><t>EEE</t></r>
+            </p>
+        '''
+
+        document = WordprocessingDocumentFactory()
+        document.add(MainDocumentPart, document_xml)
+
+        expected_html = '''
+            <p>AAABBBCCCDDDEEE</p>
+        '''
+        self.assert_document_generates_html(document, expected_html)
+
+    def test_fallback_is_a_table(self):
+        document_xml = '''
+            <p>
+                <r><t>AAA</t></r>
+                <r>
+                    <t>BBB</t>
+                    <AlternateContent>
+                        <Fallback>
+                            <tbl>
```

Comment:
> To inherit the grandparents children requires that we know what the grandchild is

It requires that the node know what it's grandparent is, which you are correct, cannot be done at the class level without some kind of special syntax. 

An additional problem is that the AlternateContent can exist as a child of any node. Currently we have it in (at least) the run node. It's not practical to implicitly add it to every node, and I'm not sure how we would ensure that it was added in every new node. 

> If we are in no rush to fix this, then I am going to punt on this.

While there's no rush, if we upgrade libreoffice without fixing this, it will mean some percentage of documents will be busted. That percentage may be low enough that it doesn't matter. You'll need to run through the QA test to see what the impact is.